Linux longtests failing: test_update_flags and test_console_output_depending_on_reportlab_existence

```python
=========================== short test summary info ============================
FAILED test/test_cli.py::TestCLI::test_update_flags - KeyError: <class 'SystemExit'>
FAILED test/test_cli.py::TestCLI::test_console_output_depending_reportlab_existence - ValueError: binary mode doesn't take an encoding argument
====== 2 failed, 19 passed, 2 skipped, 129 warnings in 446.73s (0:07:26) =======
```
The first one is probably failing due to https://github.com/intel/cve-bin-tool/commit/d4e57cbe7bca5ce81708e02b869128137e0dc787

The second I'm less sure.

--- a/cve_bin_tool/output_engine/__init__.py
+++ b/cve_bin_tool/output_engine/__init__.py
@@ -783,11 +783,12 @@ class OutputEngine:
         self.logger.info(f"{output_type.upper()} report stored at {self.filename}")
 
         # call to output_cves
-        mode = "w"
         if output_type == "pdf":
-            mode = "wb"
-        with open(self.filename, mode, encoding="utf8") as f:
-            self.output_cves(f, output_type)
+            with open(self.filename, "wb") as f:
+                self.output_cves(f, output_type)
+        else:
+            with open(self.filename, "w", encoding="utf8") as f:
+                self.output_cves(f, output_type)
 
     def check_file_path(self, filepath: str, output_type: str, prefix: str = "output"):
         # check if the file already exists

--- a/test/test_cli.py
+++ b/test/test_cli.py
@@ -178,7 +178,7 @@ class TestCLI(TempDirTest):
         )
         with pytest.raises(SystemExit) as e:
             main(["cve-bin-tool", "-u", "whatever", "-n", "json", self.tempdir])
-        assert e.value.args[0] == ERROR_CODES[SystemExit]
+        assert e.value.args[0] == 2
 
     @staticmethod
     def check_exclude_log(caplog, exclude_path, checkers):

